src/v2/Components/Artwork/Metadata.tsx

```diff
@@ -35,7 +48,7 @@ export class MetadataContainer extends React.Component<MetadataProps> {
 }
 
 export const Metadata = styled(MetadataContainer)`
-  ${garamond("s15")};
+  ${unica("s14")};
```

Comment:
cc @damassi

Reply:
We should be using `<Sans>` / palette; everything in `"v2/Assets/Fonts"` is deprecated.